rules/0575-windows_rules.xml

```diff
@@ -0,0 +1,5170 @@
+    <description>Group of SPP events</description>
+    <options>no_full_log</options>
+  </rule>
+
+  <rule id="20414" level="3">
+    <if_sid>20413</if_sid>
+    <field name="win.system.eventID">^903$</field>
+    <description>Software Protection service stopped</description>
+    <options>no_full_log</options>
+  </rule>
+
+  <rule id="20415" level="3">
+    <if_sid>20413</if_sid>
+    <field name="win.system.eventID">^16384$</field>
+    <description>Software Protection service scheduled successfully</description>
+    <options>no_full_log</options>
+  </rule>
+
+  <rule id="20416" level="3">
+    <if_sid>20413</if_sid>
+    <field name="win.system.eventID">^8197$</field>
+    <description>SLUI.exe launched</description>
+    <options>no_full_log</options>
+  </rule>
+
+  <rule id="20417" level="3">
+    <if_sid>20413</if_sid>
+    <field name="win.system.eventID">^1016$</field>
+    <description>Proof of Purchase installed successfully</description>
+    <options>no_full_log</options>
+  </rule>
+
+  <rule id="20418" level="3">
+    <if_sid>20413</if_sid>
+    <field name="win.system.eventID">^8225$</field>
+    <description>The existing scheduler data does not match the expected data</description>
+    <options>no_full_log</options>
+  </rule>
+
+  <rule id="20419" level="3">
+    <if_sid>20003</if_sid>
+    <field name="win.system.providerName">Microsoft-Windows-Security-SPP</field>
+    <field name="win.system.eventID">^8198$</field>
+    <description>License Activation (slui.exe) failed</description>
+    <options>no_full_log</options>
+  </rule>
+
+  <rule id="20420" level="4">
+    <if_sid>20001</if_sid>
+    <field name="win.system.providerName">^Application Hang$</field>
+    <field name="win.system.eventID">^1002$</field>
+    <description>Application stopped responding, program closed</description>
+    <options>no_full_log</options>
+  </rule>
+
+  <rule id="20421" level="0">
+    <if_sid>20001</if_sid>
+    <field name="win.system.providerName">^Microsoft-Windows-Search$</field>
+    <description>Group of Windows Search events</description>
+    <options>no_full_log</options>
+  </rule>
+
+  <rule id="20422" level="3">
+    <if_sid>20421</if_sid>
+    <field name="win.system.eventID">^3007$</field>
+    <description>Performance monitoring cannot be initialized</description>
+    <options>no_full_log</options>
+  </rule>
+
+  <rule id="20423" level="4">
+    <if_sid>20421</if_sid>
+    <field name="win.system.eventID">^3014$</field>
+    <description>Internal error generated by a component loaded in the system</description>
+    <options>no_full_log</options>
+  </rule>
+
+  <rule id="20424" level="4">
+    <if_sid>20421</if_sid>
+    <field name="win.system.eventID">^3024$</field>
+    <description>Cannot start update because the content sources cannot be accessed</description>
+    <options>no_full_log</options>
+  </rule>
+
+  <rule id="20425" level="5">
```

Comment:
If it is a **critical** error it cannot be level 5. At least it should be level 9.